modules/security-context-constraints-rbac.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+[id='role-based-access-to-ssc-{context}']
+= Role-based access to Security Context Constraints
+
+You can specify SCCs as a resource that is handled by RBAC. This allows
+you to scope access to your SCCs to a certain project or to the entire
+cluster. Assigning users, groups, or service accounts directly to an
+SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, specify the `scc` resource
+when creating a role.
+
+----
+oc create role <role-name> --verb=use --resource=scc --resource-name=<scc-name> -n <namespace>
+----
+
+This results in the following role definition:
+
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+...
+  name: role-name <1>
+  namespace: namespace <2> 
+...
+rules:
+- apiGroups:
+  - security.openshift.io <3>
+  resourceNames:
+  - my-permitting-scc <4>
+  resources:
+  - securitycontextconstraints <5>
+  verbs: <6>
+  - use 
+----
+<1> The role's name.
+<2> Namespace of the defined role. Defaults to `default` if not specified.
+<3> The API group that includes the SecurityContextConstraint resource. 
```

Comment:
Should "SecurityContextConstraint" be in backticks here, or should it have spaces or just be SCC?

Reply:
This has been discussed in IRC - leaving the text as is due to this being a literal API Resource.